### PR TITLE
DOC-459: Tooltips for Dashboard mysql-query-response-time

### PR DESCRIPTION
**Pull request by azhebel**

### CODE REVIEW

File: dashboards/MySQL_Query_Response_Time.json [hunk 1]
```diff
@@ -44,6 +45,7 @@
             "type": "dashboards"
         }
     ],
+    "refresh": "1m",
```
Comment (michaelcoburn) [inline dashboards/MySQL_Query_Response_Time.json]: @azhebel This was set to false before, do we need to maintain that? @askomorokhov , @rnovikovP ?

File: dashboards/MySQL_Query_Response_Time.json [hunk 2]
```diff
@@ -719,13 +760,15 @@
...
                 "tagsQuery": null,
-                "type": "query"
+                "type": "query",
+                "useTags": false
             }
         ]
     },
     "time": {
-        "from": "now-12h",
+        "from": "now-1h",
```
Comment (michaelcoburn) [inline dashboards/MySQL_Query_Response_Time.json]: @azhebel I think we should ensure this remains now-12h

File: dashboards/MySQL_Query_Response_Time.json (hunk 1 above)
Comment (askomorokhov) [inline dashboards/MySQL_Query_Response_Time.json]: Yes, we do. Here are the rules that we must follow: https://confluence.percona.com/display/SER/Grafana+dashboards+rules

Comment (michaelcoburn) [inline dashboards/MySQL_Query_Response_Time.json]: @askomorokhov ok, thanks for sharing, I wasn't aware we had a style guide! :)  Does this mean that @azhebel should re-submit his PR? Or what is our next step now to move this PR forward?

Comment (askomorokhov) [inline dashboards/MySQL_Query_Response_Time.json]: Fixed. 🙂

**Comment by askomorokhov** [comment]
Looks good!

**Pull request merged**